web/src/javascript/gameWebWorker.js

```diff
@@ -2,7 +2,9 @@ import {fetchState} from "./network";
 import {networkRefresh} from "./constants";
 
 async function updateLoop() {
+  let startTime = (new Date()).valueOf();
   postMessage(await fetchState(0));
+  setTimeout(updateLoop, 1000/networkRefresh - ((new Date()).valueOf() - startTime));
```

Comment:
Here the parentheses make sense, but I still think it would look better with a variable.